File: Modul 6/PRAK604/PRAK604-2310817310011-MuhammadRizky.py
asli = input()
palsu = input()

a = len(asli); b = len(palsu)
x = 0; y = 0

if a == b:
    for i in range (a):
        if asli[i] == palsu[i]:
            if asli[i] == ' ':
                print(" ", end = " ")

            elif asli[i] == palsu[i]:
                print("*", end = " ")
                x += 1
        
        else:
            print("#", end = " ")
            y += 1

    print("\n* = ", x)
    print("* = ", y)

    if x >= y:
        print("\nPesan Asli")
    else:
        print("\nPesan Palsu")

else:
    print("Panjang kalimat berbeda, pesan palsu")
        

File: Modul 6/PRAK604/PRAK604-2310817310011-MuhammadRizky.c
#include <stdio.h>
#include <string.h>

int main(){
    char asli[1000];
    char palsu[1000];
    scanf("%[^\n]%*c", &asli);
    scanf("%[^\n]%*c", &palsu);

    int a = strlen(asli);
    int b = strlen(palsu);

    int x = 0; int y = 0;

    if (a == b){
        for (int i = 0; i < a; i++){
            if(asli[i] == palsu[i]){
                if (asli[i] == ' '){
                    printf(" ");
                } else if (asli[i] == palsu[i]){
                    printf("*");
                    x++;
                }
                
            } else{
                printf("#");
                y++;
            }
        }

        printf("\n* = %d", x);
        printf("\n# = %d", y);

        if (x >= y){
            printf("\nPesan Asli");
        } else{
            printf("\nPesan Palsu");
        }
    } else{
        printf("Panjang kalimat berbeda, pesan palsu");
    }
    return 0;
}
